### ISSUE TITLE
Does not account for environment markers

### ISSUE THREAD

**Issue by amyreese** [python-packaging/bumpreqs#1]
In this case, bumpreqs wants to update both coverage entries to `6.4.4`, even though coverage `6.3` dropped support for Python < `3.7`.

Ran against [sphinx-mdinclude @ 93e6e422def9071b7305d799711457d0ee0ac365](https://github.com/amyreese/sphinx-mdinclude/blob/93e6e422def9071b7305d799711457d0ee0ac365/requirements-dev.txt):

```
(.venv) amethyst@mordin ~/workspace/sphinx-mdinclude main  » python -m bumpreqs --diff requirements-dev.txt
requirements-dev.txt
--- a/requirements-dev.txt
+++ b/requirements-dev.txt
@@ -1,10 +1,10 @@
 attribution==1.5.2
-black==22.1.0
-coverage==6.2; python_version < "3.7"
-coverage==6.3; python_version >= "3.7"
-flake8==4.0.1
-flit==3.6.0
-mypy==0.931
-sphinx==4.3.1
-ufmt==1.3.1.post1
-usort==1.0.1
+black==22.6.0
+coverage==6.4.4; python_version < "3.7"
+coverage==6.4.4; python_version >= "3.7"
+flake8==5.0.4
+flit==3.7.1
+mypy==0.971
+sphinx==5.1.1
+ufmt==2.0.0
+usort==1.0.4
```

**Comment by thatch** [comment]
In the case of `python_version` we can use `requires_python` from the metadata easily; do you expect other markers to be ignored (e.g. implementation pypy, platform windows, etc)

**Comment by thatch** [comment]
Fixed in 0.9.1
